runtime/trappist/src/lib.rs

```diff
@@ -290,7 +293,7 @@ impl parachain_info::Config for Runtime {}
 impl cumulus_pallet_aura_ext::Config for Runtime {}
 
 parameter_types! {
-	pub const Period: u32 = 6 * HOURS;
+	pub const Period: u32 = 10 * MINUTES;
```

Comment:
may i ask what is the purpose of this `Period` change? I don't see the `Period` type being used in the pallet_session...

Reply:
See referenced PR on the Contracts parachain.